claat/nodes/infobox_test.go

```diff
@@ -0,0 +1,109 @@
+package nodes
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+// AllowUnexported option for cmp to make sure we can diff properly.
+var cmpOpt = cmp.AllowUnexported(InfoboxNode{}, node{}, ListNode{}, TextNode{})
+
+func TestNewInfoboxNode(t *testing.T) {
+	tests := []struct {
+		name      string
+		inKind    InfoboxKind
```

Comment:
Consider dropping the `in` prefix; `kind` and `content` are sufficient.

Reply:
I think this makes it more readable, it provides more cues to what the test's inputs and outputs are.

Reply:
Parallel discussion on https://github.com/googlecodelabs/tools/pull/596#discussion_r666499324